src/web/hooks/useTranslation.ts

```diff
@@ -30,13 +32,16 @@ type TranslationHook = {
  * @returns Array|Object with the translation function, i18next instance and ready status
  */
 const useTranslation = (): TranslationHook => {
-  // provide same interface as useTranslation from i18next
-  // this will allow to use react-i18next in future if required
-  const ret = [_, i18next, true] as TranslationHook;
-  ret.t = _;
-  ret.i18n = i18next;
-  ret.ready = true;
-  return ret;
+  const [language] = useLanguage();
+
+  return useMemo(() => {
```

Comment:
Would be nice to keep the comment about the intention of the details of this hook.
```suggestion
  return useMemo(() => {
    // provide same interface as useTranslation from i18next
    // this will allow to use react-i18next in future if required
```